From 9b10bb3495648dc9ee29cba788f0f2f38f2fcdec Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Fri, 28 Jun 2024 14:10:26 +0200
Subject: [PATCH 638/841] feat(stm32mp21): DDR describe power supplies on
 STM32MP215F-DK board

Describe describe power supplies on STM32MP215F-DK board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 1561884598324a0cb116384bfc0521db47426536 feat(stm32mp21): DDR describe power supplies on STM32MP215F-DK board]
Change-Id: I1cdca8b267e5362db87e50d1a99ea05dd64ebcd8
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/390347
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp215f-dk.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/fdts/stm32mp215f-dk.dts b/fdts/stm32mp215f-dk.dts
index 54ba5b9a6..6d94d4d0b 100644
--- a/fdts/stm32mp215f-dk.dts
+++ b/fdts/stm32mp215f-dk.dts
@@ -32,6 +32,12 @@
 	};
 };
 
+&ddr {
+	vdd1-supply = <&vdd1_ddr>;
+	vdd2-supply = <&vdd2_ddr>;
+	vddq-supply = <&vdd2_ddr>;
+};
+
 &hash1 {
 	status = "okay";
 };
-- 
2.47.2

